roles/foreman_client_repositories/tasks/repo.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: 'Foreman Client Staging Repository'
+  yum_repository:
+    name: foreman-client-koji
+    description: "Foreman {{ foreman_client_repositories_version }} Client Koji Repository"
+    baseurl: "http://koji.katello.org/releases/yum/foreman-client-{{ foreman_client_repositories_version }}/el{{ ansible_distribution_major_version }}/x86_64/"
```

Comment:
`foreman_client_repositories_dists[ansible_os_family]` should also work here, right?

Reply:
probably, didn't look further into the actual repo code